engine/schema/src/main/java/com/cloud/storage/dao/SnapshotDaoImpl.java

```diff
@@ -214,26 +183,6 @@ protected void init() {
         InstanceIdSearch.done();
     }
 
-    @Override
-    public Long getSecHostId(long volumeId) {
-
-        TransactionLegacy txn = TransactionLegacy.currentTxn();
-        PreparedStatement pstmt = null;
-        String sql = GET_SECHOST_ID;
```

Comment:
The `GET_SECHOST_ID` variable is also only used here. So, what about if you delete it as well?